lib/install/vchlog/buffered_pipe.go

```diff
@@ -66,6 +69,9 @@ func (bp *BufferedPipe) Write(data []byte) (n int, err error) {
 	defer bp.c.Signal()
 
 	if bp.closed {
+		buf := make([]byte, 4096)
```

Comment:
Do make sure that these aren't checked in enabled - you can use the same `verbose` check to enable/disable the more intrusive checking if need be. Really this should be something we can hook off a trace point as a late binding thing, but...

Reply:
Definitely planning to either polish this logic (if it proves useful) or remove it before merging.